fe/fe-core/src/main/java/com/starrocks/sql/common/MetaUtils.java

```diff
@@ -78,6 +78,35 @@ public static Table getTable(long dbId, long tableId) {
         return table;
     }
 
+    public static Table getTableByUUID(String tableUUID) {
+        String[] splits = tableUUID.split("\\.");
+        if (splits.length < 3) {
+            // native table
+            Long tableID;
+            try {
+                tableID = Long.parseLong(splits[0]);
+            } catch (NumberFormatException e) {
+                throw new SemanticException("Table %s is not found", tableUUID);
+            }
+            List<Long> dbIds = GlobalStateMgr.getCurrentState().getDbIds();
+            for (Long dbId : dbIds) {
+                try {
+                    return getTable(dbId, tableID);
```

Comment:
get table from all db?

Reply:
Yes, just be compatible with native table.